io.github.bcpierce00.unison.yml

```diff
@@ -0,0 +1,88 @@
+app-id: "io.github.bcpierce00.unison"
+runtime: "org.gnome.Platform"
+runtime-version: "48"
```

Comment:
why do you need the GNOME platform?

Reply:
my assumption was that GTK3 is only contained in the Gnome runtime - but now that i check the assumption it turns out it's wrong and it is in the freedesktop runtime :)

thanks for the hint